packages/gestalt/src/Heading.js

```diff
@@ -7,6 +7,7 @@ import styles from './Heading.css';
 import typography from './Typography.css';
 
 type Props = {|
+  align?: 'left' | 'right' | 'center' | 'justify',
```

Comment:
@aiweithesushigirl could we also update the `Heading.propTypes` below? As a UI library, we need to support not just Flow but also other usages.

Reply:
Just updated!